Database/Entities/PetEnitiy.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Database.Entities
+{
+    public class PetEnitiy
+    {
+        [Key]
+        public required Guid Id { get; init; }
+
+        // foreign key 
+        public required ShelterEntity Shelter { get; init; } = null!;
+        public required Guid ShelterId { get; init; }
+
+        // fields
+        public string Name { get; init; } = null!;
+        public string Species { get; init; } = null!;
+        public string Breed { get; init; } = null!;
+        public DateTime Birthday { get; init; }
+        public string Description { get; init; } = null!;
+        
+        //public byte[] Photo { get; init; } = null!; // ?
```

Comment:
If only there was a system that would allow us to not permanently lose code once we delete it...
I guess just commenting it out is the only way...